Catappult/appcoins-wallet-android
# [APPC-3207] Vip Referral code notification

**What does this PR do?**
Notification to alert Vip users that the referral code is ready. 
The referral is checked every 5 minutes while in background, for at most 2 hours.
Uses WorkManager.

**Database changed?**
 No

**Where should the reviewer start?**
GetVipReferralWorker

**How should this be manually tested?**
Get a new wallet to vip level, make a purchase and wait for the referral notification (should take 1 minute to appear).

**What are the relevant tickets?**
https://aptoide.atlassian.net/browse/APPC-3207

**Code Review Checklist**
